database/migrations/2023_07_13_182809_dictionaries_and_positions.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+          Schema::create('dictionaries', function (Blueprint $table) {
+              $table->char('id', 36)->collation('ascii_bin')->primary();
+              $table->char('facility_id', 36)->collation('ascii_bin')->nullable();
+              $table->string('name');
+              $table->boolean('is_fixed');
+
+              $table->char('created_by', 36)->collation('ascii_bin');
+              $table->dateTime('created_at');
+              $table->dateTime('updated_at');
+
+              $table->foreign('facility_id')->references('id')->on('facilities');
+              $table->foreign('created_by')->references('id')->on('users');
+          });
+          Schema::create('positions', function (Blueprint $table) {
+              $table->char('id', 36)->collation('ascii_bin')->primary();
+              $table->char('dictionary_id', 36)->collation('ascii_bin');
+              $table->char('facility_id', 36)->collation('ascii_bin')->nullable();
+              $table->string('name');
+              $table->boolean('is_fixed');
+              $table->integer('default_order');
+
+              $table->char('created_by', 36)->collation('ascii_bin');
+              $table->dateTime('created_at');
+              $table->dateTime('updated_at');
+
+              $table->unique(['dictionary_id', 'default_order']);
+              $table->foreign('dictionary_id')->references('id')->on('dictionaries');
+              $table->foreign('facility_id')->references('id')->on('facilities');
```

Comment:
dlaczego zarówno słowniki i pozycje mają powiązanie z facility_id?

Reply:
cytując mnie: np. typ wizyty; będzie globalnym słownikiem; ale placówka będzie mogła mieć swoje pozycje w tym słowniku

Reply:
trochu mylące bo jeśli bym coś takiego zobaczył to bym pomyślał, że pozycja i słownik nad nią zawsze do tego samego facility się odwołują ;d

Reply:
jak to się nie sprawdzi, to wywalę :D

Reply:
no jakby się dało zrobić prosta strukturę, gdzie słownik danej organizacji ma wymuszone uzupełnione facility_id wszystkich pozycji,to by było lepiej, no będzie trzeba tego pilnować w kodzie